src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -217,6 +217,13 @@ func runClientProxy(cfg *config.Config) {
 	settings.Configure(cfg, version, buildDate)
 	proxiedsites.Configure(cfg.ProxiedSites)
 
+	httpClient, er := util.HTTPClient(cfg.CloudConfigCA, cfg.Addr)
+	if er != nil {
+		log.Errorf("Could not create HTTP client %v", er)
+	} else {
+		analytics.Configure(cfg, false, httpClient)
```

Comment:
I think we're calling this too soon. I'm getting ``May 19 00:55:10.898 - analytics: analytics.go:134 https://www.google-analytics.com/collect: http: error connecting to proxy http://localhost:10080: dial tcp 127.0.0.1:10080: connection refused`` -- We need to use waitforserver prior to the analytics call to make sure the client proxy is ready.